cypress/support/pages/checkout.pageObject.js

```diff
@@ -0,0 +1,10 @@
+import PageObject from "../PageObject";
+
+class CheckoutPageObject extends PageObject {
+  url = "/index.html";
+
+  clickOnButton(linkName) {
+    cy.get(".btn-success", linkName).click();
+  }
+}
+export default CheckoutPageObject;
```

Comment:
Include an empty line at the end of each file to avoid such situations on the GitHub

Reply:
Thank you, I added a line before 'export', but the empty line at the end of the file gets automatically deleted as soon as I click "Commit"